socorro/cron/jobs/bugzilla.py

```diff
@@ -186,3 +179,52 @@ def _iterator(self, from_date):
                 int(report['id']),
                 find_signatures(report.get('cf_crash_signature', ''))
             )
+
+
+class BugzillaCronAppDryRunner(App):  # pragma: no cover
```

Comment:
It looks like this is doing actual db stuff and isn't a dry run. Is that right? If so, then maybe we should rename it.